### PR TITLE
Complete rewrite and reshuffling of `Tournaments/Official support` and use British spelling in `People/The_Team` and `People/Users_with_unique_titles`

### PR DESCRIPTION
**Pull request by mangomizer**
Only thing unsure is whether we should use British English

ie. Organizers -> Organisers, Authorized -> Authorise.

- [x] Tournament committee initial review
- [x] New badge/banner examples
- [x] `SKIP_OUTDATED_CHECK`
- [x] Tournament committee final review (should be done this weekend)

### CODE REVIEW

File: wiki/Tournaments/Official_support/en.md
Comment (Bazzadwarf) [inline wiki/Tournaments/Official_support/en.md:72]:
```suggestion
Once the tournament has concluded, the tournament organizers will need to submit the following to the account support team:
```

File: wiki/Tournaments/Official_support/en.md
Comment (Bazzadwarf) [inline wiki/Tournaments/Official_support/en.md:173]: There doesn't look to be any agreed-upon name for mappool playtesters wiki-side, `mappool quality assurance team` reads too long and just `playtesters` could be kinda vague. I'll leave this up for discussion.
```suggestion
We consider referees, mappool selectors, mappool playtesters, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.
```

File: wiki/Tournaments/Official_support/en.md
Comment (Bazzadwarf) [inline wiki/Tournaments/Official_support/en.md:13]:
```suggestion
- A dedicated screening process to prevent players with serious integrity infringements (cheating, multi-accounting, tournament bans) from playing
```

**Review by Bazzadwarf — CHANGES_REQUESTED** [review]
Seems like a good improvement on the whole to me, nice work.

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [reply wiki/Tournaments/Official_support/en.md:173]: Just playtester is fine. Anyone involved with tournaments knows what it refers to. What other than mappools would you be playtesting?

**Comment by mangomizer** [comment]
Do not merge as this still requires further review from t-committee + needs new badge/banner examples

**Comment by mangomizer** [comment]
Badges/Banners added, now only waiting on further review.

**Comment by Walavouchey** [comment]
I've moved the main-menu banner image to the article folder, put the badges on top of each other similarly to how tournament pages do it, and used the regular-sized versions of the banners since they would otherwise be too big. [The ASC mandates max 800px width](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#max-image-width), but I'm unsure if that's an outdated limit. The main-menu banner example is similarly too wide, but I'll wait for someone else's opinion.

**Comment by Walavouchey** [comment]
force-pushed to remove the main-menu banner that wasn't supposed to be there from commit history

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:5]:
```suggestion
This article was last updated on January 4, 2022. For any queries and clarifications, please contact [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:22]:
```suggestion
- Staff members must not participate in the tournament themselves at any point, and registered players are unable to later assist as staff either. For our definition on "staff", please refer to the [FAQ](#what-is-considered-staff).
```

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:186]:
```suggestion
If we are going by the above definition of "staff", then the answer is no. The role they would assume could conceivably influence the outcome of the tournament. A player may conclude play in your tournament and join as a commentator, for example, but not as a referee, organizer, or anything further.
```

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:138]:
```suggestion
In rare cases, the osu!team may choose to award exceptional tournaments that go above and beyond the expected standards with badges for the top three finalists. This is done at the discretion of the osu!team—you will be contacted if your tournament is deemed worthy.
```

File: wiki/Tournaments/Official_support/en.md
Comment (LeoFLT) [inline wiki/Tournaments/Official_support/en.md:153]: This is pending confirmation, but there is a certain leniency for banner sizes

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by LeoFLT — DISMISSED** [review]
Barring the banner specifications (we're waiting on more information on that), looks good to me.

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:34]:
```suggestion
If your tournament satisfies the [eligibility criteria](#eligibility), you can make a request for support by sending an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh) with the following information:
```

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:83]:
```suggestion
Users under an active (or permanent) tournament ban are expected to disclose their tournament ban status to any badged tournament they're helping with. They may be enlisted as general help (i.e. any non-staff role such as a commentator, streamer, etc.) in an officially supported tournament at the host's discretion, but we encourage careful consideration of such choices as said individuals have already run afoul of the rules once (or more). They may not act as [staff](#what-is-considered-staff) UNLESS the host requests an exemption for them when sending in the player list for screening. The account support team will then assess these on a case-by-case basis with the user's history in mind.
```

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:119]:
```suggestion
Once screening concludes, the account support team will provide a list of any players from your tournament who failed screening and are not considered eligible for tournament play, without providing specific reasoning. Individual players who are unhappy with their screening outcome should be told to consult [accounts@ppy.sh](mailto:accounts@ppy.sh) via email.
```

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:146]:
```suggestion
- The livestreams of your current tournament average over 100 concurrent viewers.
```

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:148]:
```suggestion
- You are able to provide the schedule of the matches and livestreams ahead of time.
```

**Comment by Walavouchey** [comment]
There are some inconsistencies with the British vs American english throughout the article, where the words are mixed up. They should probably just be converted to British English

File: wiki/Tournaments/Official_support/en.md
Comment (ChillierPear) [inline wiki/Tournaments/Official_support/en.md:131]: - The badge must be **172x80**px in dimension.

**Review by ChillierPear — APPROVED** [review]
Good changes, updates significant amount of outdated material.

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [reply wiki/Tournaments/Official_support/en.md:131]: this is [already a requirement](https://github.com/ppy/osu-wiki/blob/36b83eba78ebdd178904a680b8226839994dd4df/wiki/Tournaments/Official_support/en.md?plain=1#L128) (click on collapsed lines to expand them in the pr diff)

**Review by Walavouchey — APPROVED** [review]

File: wiki/Tournaments/Official_support/en.md
Comment (mangomizer) [reply wiki/Tournaments/Official_support/en.md:153]: Confirmed. 1500x256px is the standard.
The discrepancy in size comes from scaling/edits to the banner from the osu!team, and not from user submission.

File: wiki/Tournaments/Official_support/en.md
Comment (ChillierPear) [inline wiki/Tournaments/Official_support/en.md:145]: In the above section an "experienced team member" is someone who has successfully organized/badged three or more tournaments in the past, should update this section to match those requirements.

Comment (ChillierPear) [reply wiki/Tournaments/Official_support/en.md:145]: or alternatively 
- You have successfully run and received badge support for *at least* two tournaments in the past.

**Review by ChillierPear — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:158]:
```suggestion
- It must clearly display the logo, motif, AND name of your tournament, plus any information about the ongoing stage or section that is being advertised.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/Official_support/en.md
Comment (Ephemeralis) [inline wiki/Tournaments/Official_support/en.md:27]:
```suggestion
**¹**: An "experienced" team member is loosely defined as someone who has contributed significantly to the successful running of at least **three** badge-receiving tournaments beforehand, or has been a part of a official World Cup volunteer team for any game mode. This may vary at the discretion of the account support team.
```

Otherwise, this sounds like the criteria is limited strictly to OWC helpers/volunteers, which it isn't.

File: wiki/Tournaments/Official_support/en.md
Comment (Ephemeralis) [inline wiki/Tournaments/Official_support/en.md:148]:
```suggestion
- The livestreams of your current tournament average a reasonable viewership for any game modes featured in them. More popular modes (such as osu!) will be expected to have higher average viewership in order to qualify.
```

File: wiki/Tournaments/Official_support/en.md
Comment (Ephemeralis) [inline wiki/Tournaments/Official_support/en.md:13]:
```suggestion
- Potential consideration for a [main-menu banner advertisement](#requesting-main-menu-banner-support) during major matches
```

I don't think we should list this as something explicitly "available" if that makes sense, since only a very narrow band of tournaments will ever really qualify for it. The current wording leans towards the 'this is a given' side of things and it really isn't, especially when we're so particular about what goes on the main menu.

File: wiki/Tournaments/Official_support/en.md
Comment (Ephemeralis) [inline wiki/Tournaments/Official_support/en.md:189]:
```suggestion
Staff cannot. Helpers can.
```
Chained em-dash is hideous. Never do this. Colons would have worked here as well. Super nitpicky.

**Review by Ephemeralis — CHANGES_REQUESTED** [review]
a few minor things

File: wiki/Tournaments/Official_support/en.md
Comment (Ephemeralis) [reply wiki/Tournaments/Official_support/en.md:148]: game mode equity and all that jazz

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:189]: i brought this up with mango as well but i was thinking it would make sense to move the information no [line 183](https://github.com/ppy/osu-wiki/pull/6639/files#diff-35eeda5851b226096d81386f70bacafc5f63e4b140adb48060969d2c9987c7beR183) down here since it's the only relevant information regarding this heading and explains it better.

File: wiki/People/The_Team/en.md
Comment (LeoFLT) [inline wiki/People/The_Team/en.md:70]: WalterToro/deadbeat/juankristal/Loctav are no longer actively involved in WC organization, maybe change it to  `Former World Cup organiser and administrator` or something similar?

Comment (LeoFLT) [inline wiki/People/The_Team/en.md:70]:
```suggestion
| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | Former World Cup organiser and administrator |
```

**Review by LeoFLT — CHANGES_REQUESTED** [review]
Former `World Cup organiser and administrator` probably? WalterToro/deadbeat/juankristal/Loctav are no longer actively involved in WC organization

**Comment by Ephemeralis** [comment]
I really wish the team changing stuff and unique titles stuff had been split out into its own PR. Almost zero reason for it to be bundled with a total rewrite.

TC is convening to discuss the remaining points this weekend, otherwise.

File: wiki/People/The_Team/en.md
Comment (cl8n) [reply wiki/People/The_Team/en.md:70]: this is part of the "Inactive" section

Comment (mangomizer) [reply wiki/People/The_Team/en.md:70]: agree with clayton

**Comment by mangomizer** [comment]
This is good to merge now.
Of course, guidelines can be changed at any point in the future, and we probably will be making changes in the future, but there is no point in waiting to merge the more pressing rule changes.

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:111]: let's not do both styles, pick one

```suggestion
Player1,Team1,1234567
Player2,Team2,1234567
Player3,Team3,1234567
```

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**
